sphinx/directives/__init__.py

```diff
@@ -73,15 +73,14 @@ class ObjectDescription(SphinxDirective):
     def get_field_type_map(self):
         # type: () -> Dict[str, Tuple[Field, bool]]
         if self._doc_field_type_map == {}:
+            self._doc_field_type_map = {}
             for field in self.doc_field_types:
                 for name in field.names:
                     self._doc_field_type_map[name] = (field, False)
-
```

Comment:
This PR removes unrelated empty lines. Could you fix this? Then I'll merge this soon.

Reply:
OK